tests/test_export_rms/test_export_structure_depth_isochores.py

```diff
@@ -125,16 +124,8 @@ def test_config_missing(mock_project_variable, rmssetup_with_fmuconfig, monkeypa
     """Test that an exception is raised if the config is missing."""
 
     from fmu.dataio.export.rms import export_structure_depth_isochores
-    from fmu.dataio.export.rms._utils import CONFIG_PATH
-
-    monkeypatch.chdir(rmssetup_with_fmuconfig)
-
-    config_path_modified = Path("wrong.yml")
-
-    CONFIG_PATH.rename(config_path_modified)
 
+    # move up one directory to trigger not finding the config
+    monkeypatch.chdir(rmssetup_with_fmuconfig.parent)
```

Comment:
just a small simplification of the test setup here to get rid of the `CONFIG_PATH` import